docs/articles/baseplatform/running-qemu-smoketests.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _running-qemu-smoketests:
+
+Running smoke tests on QEMU
+===========================
+
+To run smoke tests on QEMU target, proceed as follows,
```

Comment:
in QEMU

Reply:
also, it seems to be officially called "image tests": https://wiki.yoctoproject.org/wiki/Image_tests

Reply:
> in QEMU

:) I read it similar to as running "on NUC target" or "on RPI target".